setup-dev-env.sh

```diff
@@ -127,13 +127,7 @@ else
 fi
 
 # Check downloading artifacts
-if [ "$option_yes" = "true" ] || [ "$option_download_artifacts" = "true" ]; then
-    echo -e "\e[36mArtifacts will be downloaded to $option_data_dir\e[m"
-    ansible_args+=("--extra-vars" "prompt_download_artifacts=y")
-fi
-
-# Check downloading artifacts
-if [ "$target_playbook" = "autoware.dev_env.openadk" ]; then
+if [ "$target_playbook" = "autoware.dev_env.openadkit" ]; then
```

Comment:
@oguzkaganozt Please make a separate PR to review small and independent changes and merge rapidly.